client/src/pages/CalendarPage/CalendarPage.js

```diff
@@ -58,7 +58,7 @@ const CalendarPage = () => {
   return (
     <div className={styles.bg}>
       {showModal ? <TaskDetails closeModalHandler={toggleModal} /> : <></>}
```

Comment:
Unrelated to this PR but the fix can be made here: We should rename showModal to showTaskDetails instead